README.md

```diff
@@ -267,18 +267,32 @@ If you want to develop new content for this collection or improve what is alread
 
 Refer [testing](testing.md) for more information.
 
+### Updating documentation
+
+- Backup `docs/docsite` (for example `mv docs/docsite /tmp`)
+- Refresh documentation: `tox -e add_docs`
+- Restore docs/docsite (for example `mv /tmp/docsite docs`)
+
 ## Publishing New Version
 
 Prepare the release:
-- Refresh the README.md: `tox -e add_docs`
```

Comment:
Since we've got this automated now, I would just point on the playbook. This way we don't have to maintain the workflow at two different places.

Reply:
The playbook can be used for minor but not for major releases. That's why I wanted to document how to do it manually. Although... I think I see a way to make the playbook work for major updates, too. Give me some time to think this trough and work on it.